src/NfOrd/Ideal/Prime.jl

```diff
@@ -525,7 +528,7 @@ end
                        degree_limit::Int = 0, index_divisors::Bool = true) -> Array{NfOrdIdl, 1}
 
 Computes the prime ideals $\mathcal O$ with norm up to $B$.
->
+
```

Comment:
In some cases like here, there was a `>` character to separate the pargraphs, which themselves didn't have a leading `>` in their lines. I'm not sure what was the intent, but I can easily revert this (undelete `>` which separate paragraphs).

Reply:
I think these are just artifacts from a previous attempt on cleaning up the docstrings :)